cmd/swapd/main.go

```diff
@@ -78,7 +76,7 @@ const (
 	flagDeploy       = "deploy"
 	flagTransferBack = "transfer-back"
 
-	flagLog = "log"
+	flagLogLevel = "log-level"
```

Comment:
nit but could you change this back to `log` unless you feel strongly about this?

Reply:
I think `log-level` will be less confusing for our end users. I, personally, was confused not knowing if the flag was for a log file, a level, a format type like JSON, or something else.

I've worked with a lot of server programs, and most of them that let you set the log level on the command line used the flag `--log-level` or a close variation of it.

Here are some examples using `--log-level LEVEL` verbatim on the command line:
* monerod: https://monerodocs.org/interacting/monerod-reference/
* vault: https://www.vaultproject.io/docs/commands/server
* dockerd: https://docs.docker.com/engine/reference/commandline/cli/
* docker-compose: https://devopstuto-docker.readthedocs.io/en/latest/compose/commands/commands.htm
* mysql-shell: https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-application-log-introduction.html
* liquidbase: https://docs.liquibase.com/parameters/log-level.html
* gclusterd: https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3/html/administration_guide/configuring_the_log_level
* saltstack: https://docs.saltproject.io/en/latest/ref/configuration/logging/index.html